packages/spark-core/components/_accordions.scss

```diff
@@ -12,46 +12,83 @@
 }
 
 .sprk-c-Accordion__item {
-  background-color: $blue-medium;
   margin-bottom: $space-xs;
 
   &:last-child {
     margin-bottom: 0;
   }
+
+  &:first-child {
+    border-top: $accordion-border;
+  }
 }
 
 .sprk-c-Accordion--navigation .sprk-c-Accordion__item {
   background-color: $white;
   margin: 0;
+
+  &:first-child {
+    border-top: 0;
+  }
 }
 
 .sprk-c-Accordion__summary {
-  color: $white;
+  align-items: center;
+  color: $black;
   display: flex;
-  padding: $space-m;
+  padding: $space-m $space-s;
   text-decoration: none;
 
   &:hover,
   &:active {
-    background-color: $blue-dark;
-    color: $white;
+    color: $red-real;
```

Comment:
We'll want these colors set to vars and put in settings.scss so people can theme the component if needed.